src/subpackages/core/cellprofiler_core/image/abstract_image/file/_file_image.py

```diff
@@ -313,23 +465,25 @@ def __set_image(self):
 
         self.cache_file()
         channel_names = []
+
         if is_matlab_file(self.__filename):
             img = load_data_file(self.get_full_name(), loadmat)
-            self.scale = 1.0
+            # TODO - 4955: should we be overwriting if not None?
```

Comment:
So here's where you're going to punch me for not bringing this up earlier, mostly because I just didn't think about it because they aren't _images_; mat and npy were in general intended in CellProfiler to be a special case - they're intended for flatfield illumination correction images, which we do NOT actually want rescaled. We want the scaling as is, for reasons easier to explain in person (but can drop a link too if you'd rather). 

I'm not sure how useful the matlab reader is, because it requires the table have [a very specific name](https://github.com/CellProfiler/CellProfiler/blob/main/src/subpackages/core/cellprofiler_core/utilities/image.py#L129-L133), and I'm not sure if anything but CellProfiler 1 and CellProfiler 2 ever consistently made such a .mat file with such a named table. But for the numpy reader, when it is called by loading something as an illum correction function, we want to not rescale. 

(I think adding the ability to treat numpy arrays as normal images is a nice-to-have; I can point you at internal slack messages from the last few months where people in the lab wanted to be able to read npy files as images and we had to tell them "no, convert them to tiffs"; I think in the age of deep learning, this will happen more and more. But we do also need to be able to support the special case of "a numpy array being loaded as an illum correction image, which you should please not touch")

Reply:
I'm super, duper sorry for not having thought about this before!